Cargo.toml

```diff
@@ -14,9 +14,10 @@ license = "MIT/Apache-2.0"
 travis-ci = { repository = "mehcode/config-rs" }
 
 [features]
-default = ["toml", "json", "yaml"]
+default = ["toml", "json", "yaml", "hjson"]
```

Comment:
I'd rather we not add this to `default`. I'm still mulling over if `default` should be empty or not.

Reply:
I'm not sure this decision is part of this PR. At the moment, the docs implied when I read them that neither json or yaml is a default choice.

I'd suggest that formats, in the long run, are broken out into separate crates, a la serde.